test/CMakeLists.txt

```diff
@@ -1,41 +1,41 @@
 add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/IOC
-                   COMMAND mkdir ${CMAKE_CURRENT_BINARY_DIR}/IOC)
-                   
-add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/IOC/ctkTestApp/Db/
-                   COMMAND ${EPICS_PATH}/bin/${EPICS_ARCH}/makeBaseApp.pl -t ioc -u ctkTest ctkTest
-                   COMMAND ${EPICS_PATH}/bin/${EPICS_ARCH}/makeBaseApp.pl -i -t ioc ctkTest
-                   DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/IOC
-                   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/IOC)
-                   
+  COMMAND mkdir ${CMAKE_CURRENT_BINARY_DIR}/IOC)
+
+add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/IOC/ctkTestApp
```

Comment:
For the sake of completeness we should probably look for an existing Perl interpreter when building the tests.

Reply:
done